podcasts/Onboarding/OnboardingFlow.swift

```diff
@@ -87,6 +87,20 @@ struct OnboardingFlow {
         /// continue with the Sonos connection process
         case sonosLink = "sonos_link"
 
+        /// When the user was logged in but was forced logged out
```

Comment:
Wdyt about adding some more info to the comment?
```suggestion
        /// When the user was logged out due to a server or token issue, not as a result of user interaction and is
        /// asked to sign in again. See the `BackgroundSignOutListener`
```